src/test/java/com/puppycrawl/tools/checkstyle/TreeWalkerTest.java

```diff
@@ -523,6 +523,7 @@ public void testMultiCheckOrder() throws Exception {
 
         final String[] expected = {
             "6:9: " + getCheckMessage(WhitespaceAfterCheck.class, "ws.notFollowed", "if"),
+            "6:9: " + getCheckMessage(WhitespaceAroundCheck.class, "ws.notFollowed", "if"),
```

Comment:
Example of https://github.com/checkstyle/checkstyle/issues/4930#issuecomment-1399412618
> Adding ModuleName (more technically sourceClass) to comparison will mean 2 different modules who print the same violation message will no longer be seen as same and each one will be unique.